backend/src/main/java/com/redhat/cloud/notifications/routers/handlers/endpoint/EndpointResource.java

```diff
@@ -379,7 +340,7 @@ public EndpointDTO getOrCreateEmailSubscriptionEndpoint(@Context SecurityContext
     @Consumes(APPLICATION_JSON)
     @Produces(APPLICATION_JSON)
     @Operation(summary = "Add a drawer endpoint", description = "Adds the drawer system endpoint into the system and specifies the role-based access control (RBAC) group that will receive notifications. Use this endpoint to add an animation as a notification in the UI.")
-    @Authorization(legacyRBACRole = ConsoleIdentityProvider.RBAC_READ_INTEGRATIONS_ENDPOINTS, workspacePermissions = {WorkspacePermission.CREATE_DRAWER_INTEGRATION})
+    @Authorization(legacyRBACRole = RBAC_READ_INTEGRATIONS_ENDPOINTS, workspacePermissions = CREATE_DRAWER_INTEGRATION)
```

Comment:
Because this method should be removed in Q1-26, in favor of `createEndpoint` We could use `INTEGRATIONS_CREATE` as permission to validate current and future checks.

Reply:
Same than above.